app/routes/endemics/select-the-herd.js

```diff
@@ -187,8 +187,9 @@ const postHandler = {
       const { isReview } = getReviewType(typeOfReview)
 
       if (herdId === tempHerdId && typeOfReview === endemics) {
-        return h.view(endemicsSelectTheHerdException, {
+        return h.view(`${endemicsSelectTheHerdException}`, {
```

Comment:
```suggestion
        return h.view(endemicsSelectTheHerdException, {
```

Why are we wrapping the value in `${}` ?